app/subscriber/src/features/search-page/SearchPage.tsx

```diff
@@ -47,6 +46,19 @@ export const SearchPage: React.FC<ISearchType> = ({ showAdvanced }) => {
 
   const filterId = id ? parseInt(id) : 0;
 
+  const formatDate = (date: string) => {
```

Comment:
Looks like a utility function that can be extracted from the component so that it doesn't need to be created each pass.